### ISSUE TITLE
add physical_reference to CdrLocation

### ISSUE THREAD

**Issue by poohsen** [ocpi/ocpi#252]
we have it in the EVSE object and it's supposed to help visually identify an EVSE, especially in the remote start/stop case. But IMO it would also be helpful when linking back CDRs to past charging activities.

**Comment by RobertdeLeeuw-IHomer** [comment]
@poohsen , I dont understand what you are looking for. The Location and CdrLocation both already contain an  unique ID, a name and EVSE_ID. What would an extra field add?

**Comment by poohsen** [comment]
@RobertdeLeeuw-IHomer The idea is that `physical_reference` is an identifier that is visible to the driver and might have been used to select the correct EVSE when starting a session using a mobile app. It might be a value that's different from `evse.uid` or `evse.evse_id` but something the driver can relate to. That's why it would be useful to have it in a CDR.

**Comment by RobertdeLeeuw-IHomer** [comment]
Do we have an existing field in the location you would like to use?

**Comment by poohsen** [comment]
Yes. it's `location.evse.physical_reference`: "A number/string printed on the outside of the EVSE for visual identification."
Here's the PR: https://github.com/ocpi/ocpi/pull/254

**Comment by RobertdeLeeuw-IHomer** [comment]
Aaah, that one, yes, that could be very useful.
I Propose to put this on the list for the next release: OCPI 3.0 "(or if we would end up doing 2.3)

**Comment by poohsen** [comment]
@RobertdeLeeuw-IHomer had to create a new one: https://github.com/ocpi/ocpi/pull/255
